.github/workflows/release-master.yml

```diff
@@ -75,7 +75,6 @@ jobs:
       - name: Build/release Docker images
         run: | 
           docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
-          yarn build
           yarn build:docker
```

Comment:
The only thing now might be the `backend-core` module which has a similar problem as pro, where it uses a nested inner structure in dist:
```
dist/
   src/
   tests/
   package.json // this is now behind 
package.json
```

It's a bit more complicated to solve this one. It might involve moving tests inside src and updating the exposed modules definition in the `package.json` to point to the inner path instead.

Reply:
Solved in here: https://github.com/Budibase/budibase/pull/10324